auth-web/devops/vaults.env

```diff
@@ -10,6 +10,8 @@ VUE_APP_PAYMENT_PAYEE_NAME="op://relationship/$APP_ENV/auth-web/PAYMENT_PAYEE_NA
 VUE_APP_FILE_SERVER_URL="op://relationship/$APP_ENV/auth-web/FILE_SERVER_URL"
 VUE_APP_APPROVE_ACCOUNT_SLA_DAYS="op://relationship/$APP_ENV/auth-web/APPROVE_ACCOUNT_SLA_DAYS"
 VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
+VUE_APP_SOCIETIES_URL="op://web-url/$APP_ENV/bcregistry/SOCIETIES_URL"
+VUE_APP_CORP_FORMS_URL="op://web-url/$APP_ENV/bcregistry/CORP_FORMS_URL"
```

Comment:
This file is the list of 1Password keys that need to be made available as environment variables.

Please also pre-populate the .env.sample file with these keys (and values if they are not secret).